configure.ac

```diff
@@ -100,22 +100,19 @@ PKG_CHECK_MODULES([OSTREE],
 PKG_CHECK_MODULES([SOUP],
                   [libsoup-2.4])
 
-PKG_CHECK_MODULES([EOS_AUTOUPDATER],
-                  [libnm >= $NM_REQUIRED_VERSION])
-
+AX_PKG_CHECK_MODULES([EOS_AUTOUPDATER],[glib-2.0 >= $GLIB_REQUIRED_VERSION gio-2.0 gobject-2.0 libnm >= $NM_REQUIRED_VERSION ostree-1 >= $OSTREE_REQUIRED_VERSION])
```

Comment:
As far as I can tell the autoupdater does not depend on ostree

Reply:
Currently, it kind of does, in that `libeos-updater-util/util.h` included `ostree.h` in its header — so anything which used `util.h` (including the autoupdater) needed to have the `CFLAGS` for OSTree, but not actually link to it.

I’ve pushed another commit which rearranges the OSTree APIs so this isn’t necessary.